### DISCUSSION TITLE
error upgrading from 2.2.46 -> 2.2.47

### DISCUSSION THREAD

**Discussion by mrpaulr** [GNS3/gns3-gui#3584] [General]
<img width="1416" alt="Screenshot 2024-05-15 at 08 38 27" src="https://github.com/GNS3/gns3-gui/assets/60256656/d98aa275-9712-44a6-a69c-ce1ef2a7a44f">

never got any errors on previously upgrading processes on my vm.

**Comment by grossmj** [comment]
We have just released that version and the pypi packages weren't pushed yet. It should be good now.

**Reply by mrpaulr** [reply]
indeed, it's ok now.
thank you for reply.
